### ISSUE TITLE
Keyboard Shortcuts editor: does not show custom arguments

### ISSUE THREAD

**Issue by xpe** [microsoft/vscode#114935]
## Context

Keybindings may have arguments, which are kept in the `"args"` JSON key in "keybindings.json". Currently, the Keyboard Shortcuts editor does not:

* indicate the presence of custom arguments, nor
* show the contents of the arguments

## Problems

Problem #1: When using the Keyboard Shortcuts editor, a user has no way of knowing that custom arguments are in effect.

Problem #2: When using the Keyboard Shortcuts editor, if there are multiple keybindings that share a command but have different arguments, there is no way of knowing if or how they differ.

<img width="1354" alt="Keyboard Shortcuts editor showing Emmet shortcuts" src="https://user-images.githubusercontent.com/4576329/105748723-84f9a200-5f10-11eb-8772-d766793b6ce5.png">

```json
# keybindings.json
    {
        "key": "ctrl+e ctrl+w",
        "command": "editor.emmet.action.wrapWithAbbreviation",
        "when": "editorTextFocus"
    },
    {
        "key": "ctrl+e ctrl+d",
        "command": "editor.emmet.action.wrapWithAbbreviation",
        "args": {
            "abbreviation": "dfn"
        },
        "when": "editorTextFocus"
    },
```

## Recommendation

While I will grant the Keyboard Shortcuts editor may not be able to represent *all fields* from "keybindings.json", there are certainly ways to improve it.

I'd suggest adding a column showing the custom arguments.

There are certainly UX aspects to figure out. Please let me know what ideas you have. Thanks!

**Comment by xpe** [comment]
@sandy081 What is the duplicate issue? 

I looked as results from these searchs, but didn't see anything that qualifies as a duplicate:

* https://github.com/microsoft/vscode/issues?q=keybinding+args
* https://github.com/microsoft/vscode/issues?q=keybinding+arguments
* https://github.com/microsoft/vscode/issues?q=show+key+arguments
* https://github.com/microsoft/vscode/issues?q=show+key+args

If I missed something, I'm sorry. However, spending 10 minutes searching seems like a reasonable bar to meet.

In summary, I did a *lot* of search, but I see no duplicate. So, please, if you mark something as duplicate, please link which one you think is the duplicate. This can save everybody time.

**Comment by sandy081** [comment]
https://github.com/microsoft/vscode/issues/84072

**Comment by xpe** [comment]
@sandy081 Thank you. I'm glad your team is aware and tracking this. Thanks.

**Comment by xpe** [comment]
For future reference, this is a good way to search to issues labeled with "keybindings-editor":

https://github.com/microsoft/vscode/labels/keybindings-editor

This particular search finds the duplicate issue very specifically:

https://github.com/microsoft/vscode/issues?q=is%3Aopen+label%3Akeybindings-editor+args
